app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/eztwitter.html.twig

```diff
@@ -0,0 +1,41 @@
+<script>window.twttr = (function(d, s, id) {
```

Comment:
Please, replace this code with following one:
```javascript
<script>
(function (global, doc, twttr) {
    twttr = twttr || (function(d, s, id) {
        var js, 
            fjs = d.getElementsByTagName(s)[0],
            t = global.twttr || {};
        if (d.getElementById(id)) return t;
        js = d.createElement(s);
        js.id = id;
        js.src = 'https://platform.twitter.com/widgets.js';
        fjs.parentNode.insertBefore(js, fjs);

        t._e = [];
        t.ready = function(f) {
            t._e.push(f);
        };

        return t;
    }(doc, 'script', 'twitter-wjs'));

    const tweetUrl = '{{ params.tweet_url }}';
    const tweetUrlParts = tweetUrl.split('/');
    const tweetId = tweetUrlParts[tweetUrlParts.length - 1];

    twttr.ready(function () {
        twttr.widgets.load(
            twttr.widgets.createTweet(
                tweetId,
                document.querySelector('.tweet-container[data-url="{{ params.tweet_url }}"]'),
                {
                    'cards': '{{ params.cards is defined ? params.cards : '' }}',
                    'conversation': '{{ params.conversation is defined ? params.conversation : '' }}',
                    'theme': '{{ params.theme is defined ? params.theme : '' }}',
                    'linkColor': '{{ params.link_color is defined ? params.link_color : '' }}',
                    'width': '{{ params.width is defined ? params.width : '' }}',
                    'align': '{{ params.align is defined ? params.align : '' }}',
                    'lang': '{{ params.lang is defined ? params.lang : '' }}',
                    'dnt': '{{ params.dnt is defined ? params.dnt : '' }}'
                }
            )
        );
    });
})(window, window.document, window.twttr);
</script>
<div class="tweet-container" data-url="{{ params.tweet_url }}"></div>

```

Reply:
4014bf5 contains the fix according to recent internal discussion due to other issues discovered.